back/app/policies/admin_publication_policy.rb

```diff
@@ -23,6 +23,10 @@ def reorder?
     user&.active? && user.admin?
   end
 
+  def list_areas_of_projects
+    true
```

Comment:
I'm not sure about this. I'll admit to not understanding well how our policies work. This seems like a long-winded way to say 'anyone can access this', though I have seen `true` used in some other policies.